Sources_v3/StreamChatUI/ChatChannel/ChatChannelMessageComposerView.swift

```diff
@@ -161,7 +140,7 @@ open class ChatChannelMessageComposerView<ExtraData: UIExtraDataTypes>: UIInputV
     
     override open var intrinsicContentSize: CGSize {
         let size = CGSize(
-            width: superview?.bounds.width ?? super.intrinsicContentSize.width,
+            width: .zero,
```

Comment:
Why is the intrinsic width `.zero`?

Reply:
Because width is automatically calculated for width of the keyboard when using as `inputView`.